shared/version.js

```diff
@@ -0,0 +1,14 @@
+const semver = require( "semver" )
+
+function minimumVersion( request )
+{
+	if( !process.env.MINIMUM_LAUNCHER_VERSION ) return true
+	if( !request.headers["user-agent"] || !request.headers["user-agent"].startsWith( "R2Northstar/" ) ) return false
+	let v = request.headers["user-agent"].split( " " )[0].substring( 12 )
+	if( v.match( /\+dev/g ) || semver.valid( v ) && semver.satisfies( v, ">="+process.env.MINIMUM_LAUNCHER_VERSION ) ) return true
```

Comment:
Why not just `semver.gte`?

Reply:
And we don't need to check the `+dev` since it's valid semver.

Reply:
(for the record, so are RCs are with a negative so they come before)

Reply:
Resolved in 3db0cd6b71e644e4b4a088bc14f347b726a958a4